system-x/services/kudu/src/main/java/software/tnb/kudu/validation/KuduValidation.java

```diff
@@ -0,0 +1,79 @@
+package software.tnb.kudu.validation;
+
+import software.tnb.common.validation.Validation;
+import software.tnb.kudu.resource.client.KuduClient;
+
+import java.util.List;
+import java.util.Map;
+
+public class KuduValidation implements Validation {
+
+    private final KuduClient client;
+    private final List<String> mastersOnHttp, tserversOnHttp, mastersOnRpc, tserversOnRpc;
+
+    public KuduValidation(KuduClient client, List<String> mastersUsingHttp, List<String> tserversUsingHttp
+        , List<String> mastersUsingRpc, List<String> tserversUsingRpc) {
+        this.client = client;
+        this.mastersOnHttp = mastersUsingHttp;
+        this.tserversOnHttp = tserversUsingHttp;
+        this.mastersOnRpc = mastersUsingRpc;
+        this.tserversOnRpc = tserversUsingRpc;
+    }
+
+    public List<String> getMastersOnHttp() {
+        return mastersOnHttp;
+    }
+
+    public List<String> getTserversOnHttp() {
+        return tserversOnHttp;
+    }
+
+    public List<String> getMastersOnRpc() {
+        return mastersOnRpc;
+    }
+
+    public List<String> getTserversOnRpc() {
+        return tserversOnRpc;
+    }
```

Comment:
the getters do not belong to `validation`

Reply:
removed, old implementations